packages/styles/components/_c.stars-input.scss

```diff
@@ -0,0 +1,59 @@
+.mc-stars-input {
+  $parent: &;
+
+  display: flex;
+
+  &__label {
+    color: $color-star-full;
+    cursor: pointer;
+    position: relative;
+  }
+
+  &__icon {
+    @include set-stars-size($stars-medium);
+
+    fill: currentColor;

```

Comment:
using the background was not possible ? 
could we also use aria-labels instead of real labels for SR ?

Reply:
Fixed